R/rd-html.R

```diff
@@ -566,10 +566,9 @@ parse_opts <- function(string) {
 }
 
 stop_bad_tag <- function(tag, msg = NULL) {
-  abort(c(
-    paste0("Failed to parse \\", tag, "{}."),
-    i = msg
-  ))
+  bad_tag <- paste0("\\", tag, "{}")
+  msg_abort <- 'Failed to parse tag {.val {bad_tag}}.'
+  cli::cli_abort(c(msg_abort, i = msg), call = caller_env())
```

Comment:
```suggestion
  cli::cli_abort(c("Failed to parse tag {.val {bad_tag}}.", i = msg), call = caller_env())
```
?

And maybe use `call = NULL` since we're unlikely to get a useful function name here no matter what we do.